doc/source/whatsnew/v0.23.0.txt

```diff
@@ -898,6 +898,48 @@ yourself. To revert to the old setting, you can run this line:
 
   pd.options.display.max_columns = 20
 
+.. _whatsnew_0230.boxplot.sharexy:
+
+Optional sharing of x/y-axis by pandas.DataFrame().groupby().boxplot()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(:issue:`15184`)
+
+Previous Behavior:
+
+.. code-block:: jupyter-notebook
```

Comment:
See the other examples in `/pandas/plotting/_core.py` for how these should be formatted.